newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -287,6 +293,11 @@ export const GameDetailsDialog = ({
                   </Text>
                 </Line>
               </Line>
+              <Line expand justifyContent="flex-start" alignItems="center">
```

Comment:
(are we sure this doesn't create a weird empty line when none are true?)

Reply:
It adds a bit of space.
![image](https://user-images.githubusercontent.com/2611977/155287232-4df8438c-6271-469d-bb65-40a8def605f9.png)